spec/stimpack_spec.rb

```diff
@@ -1,6 +1,7 @@
 require "pathname"
 
 rails_dir = Pathname.new(File.expand_path("fixtures/rails-7.0", __dir__))
+Dir.chdir(rails_dir)
```

Comment:
Why is this necessary?

Also -- I wonder if we should have a `use_fixture` helper which loads the right rails environment. 

Separately -- I realize we might need to have separate rspec processes that load different rails environments, unless there is a way to "unload" a rails environment.

Reply:
I don't think there's a way to "unload" a rails environment. I think we should just have the entire test suite run for a single Rails (version) environment and invoke the suite multiple times.

The `Dir.chdir` is necessary because we now don't have `Rails.root` and rely on Rails directory traversal. It works for now since we don't have anything relying on the current working dir. I'll try and think of a better way to find project root...but it works for now.